docs/porting/target/spi.md

```diff
@@ -0,0 +1,133 @@
+<h2 id="spi-port">Serial Peripheral Interface (SPI)</h2>
```

Comment:
@ithinuel : This is a HAL API doc & not a user facing API doc, correct? If so, could you please add a line in the introduction that this is only for the HAL..thanks.

Reply:
This is in the target porting guide. All chapter that live here relate to the hal layer.
The other guides don't have any warning about that.
Should this be added to all other guides too ?

Reply:
Thanks @ithinuel . In that case, the warning is not required. Thank you.